context/app/static/js/components/Search/SearchWrapper.jsx

```diff
@@ -101,20 +108,36 @@ function makeTableBodyComponent(resultFields, detailsUrlPrefix, idField) {
   return function ResultsTableBody(props) {
     const { hits } = props;
     /* eslint-disable no-underscore-dangle */
+    /* eslint-disable react/no-danger */
+    /* eslint-disable jsx-a11y/control-has-associated-label */
     return (
-      <TableBody>
+      <>
         {hits.map((hit) => (
-          <StyledTableRow key={hit._id}>
-            {resultFields.map((field) => (
-              <StyledTableCell key={field.id}>
-                <a href={detailsUrlPrefix + hit._source[idField]}>{getByPath(hit._source, field)}</a>
-              </StyledTableCell>
-            ))}
-          </StyledTableRow>
+          <StyledTableBody key={hit._id}>
+            <StyledTableRow className={'highlight' in hit && 'before-highlight'}>
+              {resultFields.map((field) => (
+                <StyledTableCell key={field.id}>
+                  <a href={detailsUrlPrefix + hit._source[idField]}>{getByPath(hit._source, field)}</a>
+                </StyledTableCell>
+              ))}
+            </StyledTableRow>
+            {'highlight' in hit && (
+              <StyledTableRow className="highlight">
+                <StyledTableCell colSpan={resultFields.length}>
+                  <a
+                    href={detailsUrlPrefix + hit._source[idField]}
+                    dangerouslySetInnerHTML={{
+                      __html: hit.highlight.everything.join(' ... '),
+                    }}
+                  />
+                </StyledTableCell>
+              </StyledTableRow>
+            )}
+          </StyledTableBody>
         ))}
-      </TableBody>
+      </>
     );
-    /* eslint-enable no-underscore-dangle */
+    /* eslint-enable */
```

Comment:
```suggestion
    /* eslint-enable no-underscore-dangle react/no-danger jsx-a11y/control-has-associated-label */
```
Do we need to specify the rules for `eslint enable`?